server/db/model/Event.js

```diff
@@ -1,4 +1,73 @@
 /* eslint-disable */
 var mongoose = require('../index.js'),
     config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+    Schema = mongoose.Schema
+Location = require('./Location.js');
```

Comment:
Add location to the comma list of imports here, declaring a variable without var/let/const puts it in the global namespace